### PR TITLE
[Android] Fixing android x64 build.

### PR DESCRIPTION
**Pull request by msisov**
This pull request partly fixes android x64 build. There are some blink failures still, but I will fix them soon.

### CODE REVIEW

**Comment by rakuco** [comment]
@mrunalk

File: runtime/common/xwalk_paths.cc [hunk 1]
```diff
@@ -163,7 +163,7 @@ bool GetDownloadPath(base::FilePath* result) {
 #else
   if (!PathService::Get(DIR_DATA_PATH, result))
     return false;
-  result->Append(FILE_PATH_LITERAL("Downloads"));
+  ignore_result(result->Append(FILE_PATH_LITERAL("Downloads")));
```
Comment (mrunalk) [inline runtime/common/xwalk_paths.cc]: Is there a compiler error/warning if you don't add this? Just want to understand the rationale behind this change.

File: runtime/browser/xwalk_content_browser_client.cc [hunk 2]
```diff
@@ -430,7 +430,7 @@ XWalkContentBrowserClient::CreateThrottlesForNavigation(
         navigation_interception::InterceptNavigationDelegate::CreateThrottleFor(
             navigation_handle));
   }
-  return std::move(throttles);
+  return throttles;
```
Comment (mrunalk) [inline runtime/browser/xwalk_content_browser_client.cc]: How do you know it is preventing copy elision? Does compiler throw any error or warning message?

**Comment by mrunalk** [comment]
@msisov, LGTM. Great job moving ahead on Android. I just posted couple questions. It will be great if you can clarify those. I will go ahead and merge these changes.

**Pull request merged**